src/App.tsx

```diff
@@ -1,26 +1,22 @@
-import React from 'react';
-import logo from './logo.svg';
-import './App.css';
+import React from "react";
+import "./App.css";
 
-function App() {
+import { taskList } from "./taskList";
+
+function TaskShow() {
   return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.tsx</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+    <div>
+      {taskList.map((task) => (
+        <div>
+          <span> {task.name} </span>
+        </div>
```

Comment:
the outer div is redundant since we only need the `<span>{task.name}</span>` but it's ok for now